.travis.yml

```diff
@@ -106,7 +106,7 @@ script:
     # Test packages pararell (xargs -P)
     # Also build the docker container (with embedded build info), that will be pushed to dockerhub later on.
     - if [ "$JOB_TYPE" = compile_and_basic_tests ]; then
-        go list ./... | grep -v vendor | xargs -n1 -I {} -P 4 go test -v -covermode=atomic -coverprofile=../../../{}/coverage.txt {} ;
+        go list ./... | grep -v vendor | xargs -n1 -I {} -P 4 go test -v -covermode=atomic -coverprofile=../../../{}/coverage.txt {} || exit $? ;
```

Comment:
this is partly issue with how yaml deals with multiline strings - it's as annoying as comfortable in some cases, if you check the same config in json it's fairly easy to spot the issue: https://travis-ci.org/mendersoftware/useradm/jobs/252799952/config